src/modules/users/users.service.ts

```diff
@@ -157,6 +165,78 @@ export default class UserService {
     });
   }
 
+  public async generateNumericUniqueId(): Promise<string> {
```

Comment:
generation of ID can be UUID, but we have auto id for club id so I think clubid field is redundant

Reply:
I've changed it to UUID, but I haven't explored the generator yet. I might reuse that instead and add features to limit the ID showing up. 

https://github.com/Zeff01/formapp-be/blob/66219e291828df2daa71f8f996959c0b61301e12/src/modules/users/users.service.ts#L171